static/js/subs.js

```diff
@@ -478,9 +477,6 @@ exports.filter_table = function (left_panel_params) {
     }
 
     exports.maybe_reset_right_panel();
-
-    // this puts the scrollTop back to what it was before the list was updated again.
-    ui.get_scroll_element($(".streams-list")).scrollTop(streams_list_scrolltop);
```

Comment:
I don't understand this commit -- `ui.reset_scrollbar` sets the scrollbar to the top, whereas I think this is attempting to return us to the scroll position before the rerender (aka similar to how we rerender the message feed invisibly) so that the change is transparent to users.  Am I missing something?

Reply:
(Or is the issue that this code path _only_ runs in response to active filtering, and not background rerenders)?

Reply:
Yep, this code only runs in a few places:
* It runs when you first build the table, where it's not needed.
* It runs when you switch between the Subscribed/All tabs, which doesn't make sense. 
* It runs when you filter.  There you clearly don't want to restore the scroll-top position, as it can scroll you past the smaller list of results.  (For some reason I can't reproduce this, which makes me think the code is just completely broken.)
* It runs when you sort.

The sensible thing here would be to have something like `rescroll_active_element_back_to(original_position_of_active_element)`, but that's not what this code is doing.

The last substantive change to this code was 141088586b8f33dcaa0ed0ff6d52a6d8c5ae09a4 (go to SimpleBar).

e0a9310c41b998a3f144188df32d1fab3e3a66d1 from Brock is the commit that introduced it.  You can verify on master that what we actually do is just prepend the stream to the top of the list, and we don't go through this codepath.

Reply:
Ahh, OK, so this is probably something where we originally did Brock's commit, and then later decided on the "place at top" strategy as better, and didn't remove the original scroll logic.